config/config_generator/main.go

```diff
@@ -24,19 +24,21 @@ func main() {
 
 	args := flag.Args()
 	if len(args) < 1 {
-		fmt.Println("Input file is missing.")
-		os.Exit(1)
+		fmt.Println("Output file name is missing.")
+		usage()
 	}
-
-	configStore, err := config.NewFileStore("config.json", true)
-	if err != nil {
-		panic(err)
+	outputFile := args[0]
+	if f, err := os.Open(outputFile); err == nil {
```

Comment:
Suggest using [os.Stat](https://golang.org/pkg/os/#Stat) and [os.IsNotExist](https://golang.org/pkg/os/#IsNotExist) to avoid having to clean up the file resource.